misk-hibernate/src/main/kotlin/misk/hibernate/pagination/RealPager.kt

```diff
@@ -39,4 +41,6 @@ internal class RealPager<T : DbEntity<T>, Q: Query<T>>(
       contents to false
     }
   }
+
+  override fun hasNext() = hasMore
```

Comment:
Can we pick one phrasing and use it uniformly?

Reply:
changed